Derpitcraft/README.md

```diff
@@ -1,105 +1,65 @@
-
 #### Version support
-Minecraft **Bedrock Edition 1.18.0 - 1.18.10** and Minecraft **Java Edition 1.18.x**.
-
+Minecraft **Bedrock Edition** `1.18.0 - 1.18.10` and Minecraft **Java Edition** `1.18.x`.
 #### How to join
-Run `$ip` to find the IPs for both bedrock and java. Join the **TrixMC Network** using the IPs. **Right click** on the **Derpitcraft Season 9** NPC to join. You can join the SMP after **April 20th 2022**.
-
+Run `$ip` on our discord to find the IPs for both bedrock and java. Join **TrixMC** using the IP. **Right-click**  the **Derpitcraft Season 9** NPC to join (If you are on mobile, you can tap and hold the NPC.). You can join the SMP after **April 20th 2022**.
 #### Spawn coordinates
-
 **Coordinates [X, Z]:**: `[-161, -223]` 
-
-This is world center. There is a world border at **7500 blocks**.
-
+This is the world centre. There is a world border at **7500 blocks** on each axis.
 #### Shopping District coordinates
-
 **Coordinates [X, Z]:**: `[-250, -262]`
-
-Scroll down to read more about what the Shopping District is (**Part 5**).
-
+Scroll down to read more about what the Shopping District is (**Point 5**).
 ## Rules and Guidelines
-
 ### Rule Of Thumb
-
 All the following rules can be summarised simply, in one line. "Be kind to others on the server". This server follows a Honor Code, and we expect everyone to respect each other. Disrespect is actionable upon. We want to ensure a safe and friendly experience for everyone, and this is not possible without your co-operation. We kindly request you to follow all these rules and keep everything playful and fun.
-
 ### Detailed Rules and Guidelines
-
-1. **No Griefing Policy**: Griefing is not tolerated at all. If you are caught, you will be **banned** from Derpitcraft Season 9 indefinitely. Our moderators will decide a suitable punishment. Griefing also includes stealing items from other players, destroying their bases, structures, and farms, and causing deaths / killing players (The PvP policy is explained in **Part 2**). It includes *harmful* pranking. Read more about the pranking policy in **Part 3**. For a better idea on what consists of a player's base, structure, or farm, see **Part 4**.
-
-  > To report griefing, or a player in any other condition, read the **Reporting** section of the **How To Guide**.
-
+1. **No Griefing Policy**: Griefing is not tolerated at all. If you are caught, you will be **banned** from Derpitcraft Season 9 indefinitely. Our moderators will decide a suitable punishment. Griefing also includes stealing items from other players, destroying their bases, structures, and farms, and causing deaths / killing players (The PvP policy is explained in **Point 2**). It includes *harmful* pranking. Read more about the pranking policy in **Point 3**. For a better idea on what consists of a player's base, structure, or farm, see **Point 4**.
+To report griefing, or a player in any other condition, read the **Reporting** section of the **How To Guide**.
```

Comment:
Markdown won't render this separately, add new line between paragraphs, also indent in to make markdown understand its part of the point.

```suggestion

    To report griefing, or a player in any other condition, read the **Reporting** section of the **How To Guide**.
```

Reply:
if it renders correctly please attach a screenshot im not sure it will